UPGRADING.md

```diff
@@ -53,6 +53,44 @@ With the setup complete, please review the Breaking Changes section and resolve
 
 See [PR #8727](https://github.com/activeadmin/activeadmin/pull/8727) for more details.
```

Comment:
I can't select it since it's partially out of scope but can you please delete lines 49-55 since they have been moved below to the upgrading-from-earlier-4.x-beta section? It's best all contained there.